tasks/Node/scripts/clean.js

```diff
@@ -0,0 +1,47 @@
+var fs = require('fs');
+var path = require('path');
+var rimraf = require('rimraf');
+
+var distDir = path.resolve(__dirname, '../dist');
+var outDir = path.resolve(__dirname, '../out');
+var azureDir = path.resolve(__dirname, '../node_modules/@azure');
+
+function clean(destDir) {
+  rimraf(destDir, function (err) {
+    if (err) {
+      console.error('Error cleaning directory: ' + destDir);
+      process.exit(1);
+    }
+
+    console.log('Cleaned ' + destDir);
+  });
+}
+
+function cleanModules(modulesDir) {
```

Comment:
Not sure why this command doesn't use rimraf. Is this leftover from when you installed node_modules within each task folder prior to the webpack bundling?

Reply:
It is meant to clean in subdirectories. I didn't know how to accomplish it with `rimraf`, I suppose. If the function can be converted, I'd be happy to. For now, it does what's needed. I call `clean` which does use `rimraf`.